### PR TITLE
Add timeout before make backup

### PR DESCRIPTION
**Pull request by stgleb**
Bring wait for IO back after 

https://github.com/portworx/torpedo/commit/4a9fc3d25e3e55d01ecd66cf32d521923a9f0009#diff-3ca1d6fd1651a59da2ec30fed2263e9e

test job

https://jenkins.portworx.co/job/Torpedo/job/gleb-eks-backup/34/

### CODE REVIEW

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -188,6 +188,7 @@ var _ = Describe("{BackupCreateKillStoreRestore}", func() {
 			//ValidateApplications(contexts)
 		})
 
+		time.Sleep(time.Minute * 20)
```
Comment (harsh-px) [inline tests/backup/backup_test.go:191]: @vinayakshnd can you sync up with Gleb on why the validate applications was commented above?

Comment (stgleb) [reply tests/backup/backup_test.go:195]: Sure, main goal of that sleep is wait for IO to happen.

**Comment by vinayakshnd** [comment]
@stgleb as discussed we are going to use ValidateApplications with skipping volumes and remove sleep here, right?

**Comment by stgleb** [comment]
@vinayakshnd yeah, today I've found this is more serious issue than I thought initially. I think all fixes will go in eks-specs PR.

**Comment by thiguetta** [comment]
Why do we decide to reopen and merge? @stgleb @vinayakshnd

**Comment by stgleb** [comment]
Sinc e #467 is stuck for a while we need this chages to be merged to unblock tx-next-backup job.

**Pull request merged**